var/spack/repos/builtin/packages/rocm-validation-suite/package.py

```diff
@@ -55,5 +57,7 @@ def cmake_args(self):
         return [
             self.define('HIP_INC_DIR', self.spec['hip'].prefix),
             self.define('ROCM_SMI_DIR', self.spec['rocm-smi-lib'].prefix),
-            self.define('ROCBLAS_DIR', self.spec['rocblas'].prefix)
+            self.define('ROCBLAS_DIR', self.spec['rocblas'].prefix),
+            self.define('YAML_INC_DIR', self.spec['yaml-cpp'].prefix.include),
+            self.define('YAML_LIB_DIR', self.spec['yaml-cpp'].prefix.lib64)
```

Comment:
lib64 is not guaranteed to be the right directory, see other packages that use `libs.directories`